packages/client/src/clients/in-app/message-client.ts

```diff
@@ -0,0 +1,98 @@
+import Knock from "../../knock";
+import { NetworkStatus, isRequestInFlight } from "../../networkStatus";
+
+import { InAppChannelClient } from "./channel-client";
+import {
+  FetchInAppMessagesOptions,
+  InAppMessageClientOptions,
+  InAppMessagesQueryInfo,
+} from "./types";
+
+const defaultOptions: Pick<InAppMessageClientOptions, "archived"> = {
+  archived: "exclude",
+};
+
+/**
+ * Manages realtime connection to in app messages service.
+ */
+export class InAppMessageClient {
+  private knock: Knock;
+  private defaultOptions: InAppMessageClientOptions;
+
+  constructor(
+    readonly channelClient: InAppChannelClient,
+    readonly messageType: string,
+    options: InAppMessageClientOptions,
```

Comment:
```suggestion
    options: InAppMessageClientOptions = {},
```
Nit, can fall back to an empty option obj, so you don't have to pass in `{}` at a constructor call?